src/resources/order/order.route.js

```diff
@@ -4,7 +4,7 @@ import { verifyAuth, validateAdmin } from '../../middlewares/validateUserAuth';
 import {
   addOrder,
   updateOrder,
-  getOrderById,
+  // getOrderById,
```

Comment:
Why are you commenting this out?

Reply:
mistake

Reply:
I moved the route to users folder and forgot to remove it